### PR TITLE
Lu 3793 save validation output

### PR DESCRIPTION
**Pull request by ranjit-das**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -46,23 +46,25 @@ package:
...
       INPUT_QUEUE_URL: ${file(./serverless-config.json):INPUT_QUEUE_URL}
       ERROR_QUEUE_URL: ${file(./serverless-config.json):ERROR_QUEUE_URL}
-    # vpc:
-    #   securityGroupIds:
-    #     - ${file(./config.dev.json):SECURITY_GROUP_ID}
-    #   subnetIds:
-    #     - ${file(./config.dev.json):SUBNET_ID}
+      BUSINESS_LAYER_ENDPOINT: ${file(./serverless-config.json):BUSINESS_LAYER_ENDPOINT}
+
+    vpc:
```
Comment (dllmorgan) [inline serverless.yml]: Hardcoded VPC IDs

Comment (ranjit-das) [reply serverless.yml]: Moved VPC related info to serverless-config.json

File: serverless.yml [hunk 2]
```diff
@@ -46,23 +33,23 @@ package:
...
+      securityGroupIds:
+        - ${file(./serverless-config.json):SECURITY_GROUP_ID}
+      subnetIds:
+        - ${file(./serverless-config.json):SUBNET_ID}
     events:
       - sqs:
          arn: ${file(./serverless-config.json):INPUT_QUEUE_ARN}
          batchSize: 1
-    timeout: 30
+    timeout: 3
```
Comment (RyanAldy) [inline serverless.yml]: Maybe increase the timeout?

File: lambda_function.py [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+        request_response = requests.post(business_layer_endpoint, json.dumps(data_to_business_layer))
         print('Response: ' + str(request_response))
         print(request_response.text, "TEXT")
         print(request_response.content, "CONTENT")
         print(request_response.status_code, "STATUS CODE")
-    except:
-        request_response = ''
-        print("{\"Error\": \"Problem with call to Business Layer\"}")
+    except Exception as error:
+        # request_response = ''
```
[the remark this answers is not in the document]
Comment (ranjit-das) [reply lambda_function.py]: Deleted the commented code.

File: serverless.yml (hunk 2 above)
Comment (ranjit-das) [reply serverless.yml]: Considering the need to run the whole flow within a few seconds, I thought to put 3 seconds in each lambda. Also it requires to be less or equal to the input queue's visibility timeout. To find its right value, need to investigate it with real data and business requirements of processing time of validation flow.

Comment (ranjit-das) [reply serverless.yml]: As discussed increased to 15 seconds.

**Pull request merged**